test/server.js

```diff
@@ -0,0 +1,13 @@
+const http = require('http')
+
+const port     = 8000
+const hostname = '0.0.0.0'
+
+http.createServer((req, res) => res.end(`
+Hello World, from the future!
+It's ${
+  new Date().getFullYear() + Math.ceil(Math.random() * 100)
+    } here, how is it going back there in ${new Date().getFullYear()}? :)
```

Comment:
At the time, I thought  "Why not. It's just a test..."
Though a more serious comment could be arranged, if it bothers anybody :smiley: